spec/factories/ext_management_system.rb

```diff
@@ -18,6 +18,12 @@
   factory :ems_container, :aliases => ["manageiq/providers/container_manager"], :class => "ManageIQ::Providers::ContainerManager", :parent => :ext_management_system do
   end
 
+  factory :ems_middleware,
+          :aliases => ["manageiq/providers/middleware_manager"],
+          :class   => "ManageIQ::Providers::MiddlewareManager",
+          :parent  => :ext_management_system do
+  end
```

Comment:
This is much nicer to read!  We should probably change the rest of these factories to match (not in this PR, but in a separate PR)